spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -203,6 +212,43 @@ private SpoonSummary runTests(AndroidDebugBridge adb, Set<String> serials) {
     return summary.end().build();
   }
 
+  /** Returns a {@link Runnable} to launch the script between devices in sequential mode. */
+  private Runnable getRunnableScript() {
+    return new Runnable() {
+      @Override
+      public void run() {
+        executeScript();
+      }
+    };
+  }
+
+  /** Execute the script file specified in param --script */
+  private void executeScript() {
+    if (script != null && script.exists()) {
+      try {
+        Runtime run = Runtime.getRuntime();
+        Process proc = run.exec(new String[] {
+                "/bin/bash", "-c", "echo 5 | " + script.getAbsolutePath()
```

Comment:
That's an excellent question, I did not see that XD. I think it was for test purposes ^^
I'm gonna change that and rename the feature too (probably tomorrow).

Thank you.

Olivier